docs/fsharp/getting-started-netcore.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Building simple Fsolutions with the .NET CLI
+description: Building simple Fsolutions with the .NET CLI
+keywords: .NET, .NET Core
+author: cartermp
+manager: wpickett
+ms.date: 06/20/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: dotnet
+ms.assetid: 615db1ec-6ef3-4de2-bae6-4586affa9771
+---
+
+Getting started with F# on .NET Core
+====================================
```

Comment:
nit: we're standardizing on the # for headers. it supports more levels and is what the other teams are using. but no big deal to keep it